IPython/core/completer.py

```diff
@@ -142,14 +142,11 @@
 # skip module docstests
 skip_doctest = True
 
-try:
-    import jedi
-    jedi.settings.case_insensitive_completion = False
-    import jedi.api.helpers
-    import jedi.api.classes
-    JEDI_INSTALLED = True
-except ImportError:
-    JEDI_INSTALLED = False
+import jedi
```

Comment:
Yup, this is dangerous. Perhaps you can revert to the old way temporarily?

Reply:
It's not that dangerous, since Jedi is an official IPython dependency (https://github.com/ipython/ipython/blob/5073312daa51b62aceb06626cf265d2da1836c7a/setup.py#L188)

For this PR I've opted for the "go fast break things" method. It'll require lots of testing on various frontends, as Carreau said above, but at least it will clean this up quickly.